R/nullModelApproxSE.R

```diff
@@ -0,0 +1,217 @@
+## estimate the ratio (se.ratio) of the true SE(score) to the fast SE(score)
+## update the null model to contain this parameter
+## this needs to be run before using the approx.score.se option in assocTestSingle
+## this is based off of the SAIGE variance approximation
+
+
+setGeneric("fitNullModelApproxSE", function(x, ...) standardGeneric("fitNullModelApproxSE"))
+
```

Comment:
@smgogarten i'm not sure if all of these input arguments need to be specified in the method here or not. i always get a bit confused about that.